apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -28,51 +29,72 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	);
 
 	return (
-		<section className={styles["signup-container"]}>
-			<div className={styles["logo-container"]}>
-				<div
-					className={getValidClassNames(
-						styles["circle"],
-						styles["circle-pink"],
-					)}
-				/>
-				<h1>LOGO</h1>
-			</div>
-			<h3 className={styles["title-form"]}>CREATE AN ACCOUNT</h3>
-			<span className={styles["auth-info"]}>
-				Already have an account? Go to
-				<Link to={AppRoute.SIGN_IN}> Sign in</Link>
-			</span>
-			<form onSubmit={handleFormSubmit}>
-				<div className={styles["input-groups"]}>
+		<div className={styles["container"]}>
+			<div className={styles["form-container"]}>
+				<form className={styles["form"]} onSubmit={handleFormSubmit}>
+					<div className={styles["form-header"]}>
+						<div className={styles["form-header__logo-container"]}>
+							<div className={styles["form-header__logo"]} />
+							<span className={styles["form-header__logo-text"]}>LOGO</span>
+						</div>
+
+						<h1 className={styles["form-header__text"]}>CREATE AN ACCOUNT</h1>
+						<span className={styles["form-header__sub-text"]}>
+							Already have an account? Go to{" "}
+							<Link to={AppRoute.SIGN_IN}>Sign in</Link>
+						</span>
+					</div>
+
 					<Input
 						control={control}
 						errors={errors}
-						label="Name"
+						label="name"
 						name="name"
 						placeholder="name"
 						type="text"
 					/>
+
 					<Input
 						control={control}
 						errors={errors}
 						label="Email"
 						name="email"
-						placeholder="email"
+						placeholder="name@gmail.com"
 						type="email"
 					/>
+
 					<Input
 						control={control}
 						errors={errors}
 						label="Password"
 						name="password"
-						placeholder="password"
+						placeholder="*******"
 						type="password"
 					/>
-				</div>
-				<Button label="CREATE AN ACCOUNT" type="submit" variant="dark" />
-			</form>
-		</section>
+
+					<Button label="CREATE AN ACCOUNT" type="submit" variant="dark" />
+				</form>
+			</div>
+
+			<div className={styles["logo-container"]}>
+				<span className={styles["logo"]}>LOGO</span>
+			</div>
+
+			<img
+				alt="ripple-effect-bg"
```

Comment:
<img width="786" alt="image" src="https://github.com/user-attachments/assets/7655ec6c-440d-4b77-ab63-123dd4003a95">
Please, update the `alt` to the meaningful text or remove this attribute.